tests/unit/tut/tut.hpp

```diff
@@ -235,7 +235,8 @@ class test_group : public group_base, public test_group_posix
                         " an SEH exception");
                 }
 #else
-                bool d = delete_obj();
+                bool d;
+                d = delete_obj();
```

Comment:
Nitpick, but can you add an inline comment that explains this? Otherwise I'll be tempted to change it back when clang-tidy or some other thing complains about this version.

Reply:
Disregard, I did it myself.